PicApp_git/PicApp/routes/web.php

```diff
@@ -16,14 +16,16 @@
 Auth::routes();
 
 Route::group(['middleware' => 'auth'], function () {
+    // 投稿一覧画面
     Route::get('/top', 'PostsController@index')->name('top');
-    Route::post('/posts', 'PostsController@store')->name('posts');
-
+    //新規投稿画面
     Route::get('/posts', 'PostsController@showCreateForm')->name('post.create');
-
+    //投稿追加処理
+    Route::post('/posts', 'PostsController@store')->name('posts');
+    //投稿詳細画面
     Route::get('/posts/{post_id}', 'PostsController@showPostDetail')->name('detail');
-
+    //投稿詳細へコメント処理
     Route::post('/posts/{post_id}/comments', 'PostsController@storeComment');
-
-    Route::post('/comments/{comment_id}', 'CommentsController@destroy');
+    //投稿削除処理
+    Route::get('/posts_delete/{post_id}', 'PostsController@destroy');
```

Comment:
getはデータを取得するようなルーティングに使うのが正しいので、データを削除する場合は `delete` を使うようにしましょう
そしてHTTP METHODをdeleteにすることで、URL名にdeleteを入れる必要がなくなります。
```suggestion
    Route::delete('/posts/{post_id}', 'PostsController@destroy');
```

またこの変更をするとView側もGETではなくDELETEとしてリクエストを送る必要が出てくるので
ネットで調べて改修をしてみましょう。